src/components/Comparison.tsx

```diff
@@ -0,0 +1,225 @@
+import React from "react"
+import styled from "styled-components"
+import CheckIcon from "@material-ui/icons/Check"
+import CloseIcon from "@material-ui/icons/Close"
+import MapIcon from "@material-ui/icons/Map"
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Typography,
+} from "@material-ui/core"
+import ActionButton from "components/Button"
+import { VerticalSpacer } from "components/Spacer"
+import { bps } from "theme/index"
+
+const StyledTableContainer = styled(TableContainer)`
+  background-color: ${props => props.theme.palette.background.paper};
+  color: ${props => props.theme.palette.text.hint};
+  width: 70%;
+  margin: 0px auto 50px auto;
+  border-radius: 8px;
+  padding: 0px 16px;
+  ${bps.down("md")} {
+    width: 80%;
+  }
+  ${bps.down("xs")} {
+    width: 90%;
+  }
+`
+
+const StyledTableCell = styled(TableCell)`
+  color: ${props => props.theme.palette.text.hint};
+  height: 40px;
+`
+
+const ComparisonTable = () => {
+  function createData(
+    Features: string,
+    KintoHub: any,
```

Comment:
action: use `React.ReactNode` instead of `any`. You should try to avoid `any` unless it is some `json` object that you cannot define the type